tensorflow_lite_support/ios/task/processor/sources/TFLClassificationResult.m

```diff
@@ -21,7 +21,7 @@ - (instancetype)initWithHeadIndex:(NSInteger)headIndex
   self = [super init];
   if (self) {
     _headIndex = headIndex;
-    _categories = categories;
+    _categories = [categories copy];
```

Comment:
After the benchmark I did when designing the ImageSegmenter output, I learnt that memory copy here can become quite expensive. So to avoid excessive copy, let's keep the strong reference of the value as you're doing now rather than copy them.

Reply:
Fixed this. reverted for all string and array copies